lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +23,168 @@ fields:
+    required: 'No'
+    type: dateTime
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.authored
+    description: The date and/or time that this set of answers were last changed.
+    action: update
+    example: |
+      {
+        "authored": "2019-10-25T14:57:43Z"
+      }
+
+  - name: item
+    required: 'Yes'
+    type: List of BackboneElement
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item
+    description: A group or question item from the original questionnaire for which answers are provided.
+    action: update
+    example: |
+      {
+        "item": [
+          {
+            "linkId": "93",
+            "text": "Tobacco",
+            "item": [
+              {
+                "linkId": "93-123",
+                "text": "Number of Years:",
+                "answer": [
+                  {
+                    "valueQuantity": "20" 
+                  }
+                ]
+              }
+            ] 
+          }
+        ]
+      }
+
+  - name: item.linkId
+    required: 'Yes'
+    type: string
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId
+    description: The item from the Questionnaire that corresponds to this item in the QuestionnaireResponse resource.
+    action: update
+    example: |
+      {
+        "linkId": "93"
+      }
+
+  - name: item.text
+    required: 'No'
+    type: string
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.text
+    description: Text that is displayed above the contents of the group or as the text of the question being answered.
+    action: update
+    example: |
+      {
+        "text": "Tobacco"
+      }
+
+  - name: item.item
+    required: 'No'
+    type: List of BackboneElement
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.item
+    description: Questions or sub-groups nested beneath a question or group.
+    action: update
+    example: |
+      {
+        "item": [
+          {
+            "linkId": "93-123",
+            "text": "Number of years:",
+            "answer": [
+              {
+                "valueQuantity": "20"
+              }
+            ]
+          }
+        ]
+      }
+
+  - name: item.item.answer
+    required: 'Yes'
+    type: string | Coding | Quantity
+    url: https://www.hl7.org/fhir/r4/resource-definitions.html#Resource.id
+    description: The respondent's answer(s) to the question.
+    action: update
+    example: |
+      {
+        "answer": [
+          {
+            "valueString": "Text"
+          }
+        ]
+      }
+    example2: |
+      {
+        "answer": [
+          {
+            "valueCoding": {
+              "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/nomenclature",
+              "code": "960439",
+              "display": "Yes"
```

Comment:
should we include display here?

Reply:
I removed it, we can add it back if needed, but I don't think that's necessary: 3c70eb3e8f903fb3e8aa8f3c2008e285b1d9705b

Reply:
Gotcha, ok it's removed from this file but added back to the examples read/search: 498b52be01eefb92bff57ef619ce7f6e8c629f29

Reply:
Missed a comman (cd8e81758a821a37e435497725b01c3e4176815b)